sdk/servicebus/service-bus/src/serializers/queueResourceSerializer.ts

```diff
@@ -41,19 +44,19 @@ export function buildQueueOptions(queueOptions: QueueOptions): InternalQueueOpti
     DuplicateDetectionHistoryTimeWindow: queueOptions.duplicateDetectionHistoryTimeWindow,
     MaxDeliveryCount: getStringOrUndefined(queueOptions.maxDeliveryCount),
     EnableBatchedOperations: getStringOrUndefined(queueOptions.enableBatchedOperations),
-    SizeInBytes: getStringOrUndefined(queueOptions.sizeInBytes),
-    MessageCount: getStringOrUndefined(queueOptions.messageCount),
     AuthorizationRules: getRawAuthorizationRules(queueOptions.authorizationRules),
+    Status: getEntityStatusOrUndefined(queueOptions.status),
+    ForwardTo: getStringOrUndefined(queueOptions.forwardTo),
```

Comment:
Has this been changed in #6382 as well?

Reply:
Changes from #6382 will override the appropriate changes and should be taken care of during merge.
Here we look at the ordering whereas in the other PR it's about handling and setting the values correctly.